src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -61,7 +64,7 @@ private PartModuleFields GetModule(string modName)
             foreach (PartModule mod in Part.Modules)
             {
                 SafeHouse.Logger.Log(string.Format("Does \"{0}\" == \"{1}\"?", mod.moduleName.ToUpper(), modName.ToUpper()));
```

Comment:
We should remove these logs.  They were there from when I was debugging this.  It's probably the main reason GetModule is slow, since using Unity's logger is a very expensive operation (it basically re-opens the file on each operation, then closes it again, rather than leaving the file open through the life of the program.  This is done so that logged messages exist immediately in the file with no buffering, so that in a crash you have an accurate picture of how far the program got).